src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNet/AspNetMvcIntegration.cs

```diff
@@ -110,19 +110,16 @@ public static Scope CreateScope(ControllerContextStruct controllerContext)
                 var tracer = Tracer.Instance;
                 var tagsFromHeaders = Enumerable.Empty<KeyValuePair<string, string>>();
 
-                if (tracer.ActiveScope == null)
```

Comment:
So if I understand this change correctly, previously we were only extracting the tags from headers when there was a propagated distributed context, and this change ensures we always extract the headers, even when there's no distributed tracing?

Reply:
It's slightly different than what you stated. Before, we would only parse the distributed tracing headers + header tags when this was a local root span (first web server span in the trace). I modified this so that we extract the header tags for each web server integration, regardless if it's a root span. This adds additional processing when we have a `aspnet.request` span followed by a `aspnet_mvc.request` span. This change isn't strictly necessary to achieve the overall feature since we only need the root span to get the header tags. I'd be okay with removing this and reverting back to the previous implementation. Do you think I should go ahead and remove it?

Reply:
I went ahead and reverted the behavior so we only do this once on the root span, just as before. Let me know if you feel strongly one way or the other, but it's probably best to do as few operations as possible. Commit: 7e13cfd3c8bfce55ccc70edc83c1abe3889ee5ba